packages/product-react/package.json

```diff
@@ -9,7 +9,7 @@
     "dev": "yarn storybook",
     "build": "gulp scripts:es scripts:umd scripts:compiled",
     "test": "jest",
-    "storybook": "start-storybook -p 6007",
+    "storybook": "start-storybook -p 6008",
```

Comment:
Why the port change?

Reply:
oh both product-react and core-react used the same port `6007` - was that on purpose? I had to update when trying to run them all at the same time